products/mxlinux.md

```diff
@@ -0,0 +1,83 @@
+---
+permalink: /mxlinux
+alternate_urls:
+-   /mx_linux
+title: MX Linux
+category: os
+releasePolicyLink: https://mxlinux.org/release-cycle/
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
+releasePolicyLink: https://mxlinux.org/release-cycle/
+purls:
+-   purl: pkg:os/mxlinux
+auto:
+-   distrowatch: devuan
+    regex: '^Distribution Release: MX Linux (?P<major>\d)\.(?P<minor>\d)$'
+releases:
+-   releaseCycle: "21"
+    codename: "Wildflower"
+    releaseDate: 2021-10-21
+    latest: "21.2.1"
```

Comment:
This is missing on the Distrowatch page, and our automation will overwrite this with `12.2`. We can either keep updates disabled, or get it corrected at distrowatch.